docs/digger/installation/installer-variables.asciidoc

```diff
@@ -0,0 +1,311 @@
+== Variables
+
+Tables below shows all variables used in the installer per role.
+
+=== android-sdk
+
+|===
+| Variable name | Description | Default value
+
+|android_sdk_home
+|Location of Android SDK that is installed in the Android SDK container.
+|/opt/android-sdk-linux
+|===
+
+=== configure-buildfarm
+
+|===
+| Variable name | Description | Default value
+
+|project_name
+|Name of the OpenShift project.
+|jenkins
+
+|concurrent_android_builds
+|Number of max number of concurrent Android builds that are allowed.
+|5
+|===
+
+
+=== deploy-jenkins
+
+|===
+| Variable name | Description | Default value
+
+|project_name
+|Name of the OpenShift project.
+|jenkins
+
+|enable_oauth
+|Whether to enable OAuth OpenShift integration. If false, a static account will be initialized.
+|false
+
+|master_memory_limit
+|Maximum amount of memory the Jenkins master container can use.
+|3Gi
+
+|master_volume_capacity
+|Volume space available for data.
+|40Gi
+
+|===
+
+=== deploy-nagios
+
+Variables below are used in Nagios for sending alert emails.
+
+|===
+| Variable name | Description | Default value
+
+|smtp_server
+|SMTP server to send alert emails.
+|localhost
+
+|smtp_username
+|SMTP username.
+|username
+
+|smtp_password
+|Password of the SMTP user.
+|password
+
+|smtp_from_address
+|SMTP from address.
+|admin@example.com
+
+|rhmap_admin_email
+|Destination address of alert emails.
+|root@localhost
+
+|===
+
+
+=== java
+
+Variables below are to configure the JDK to install on the remote that runs Jenkins CLI commands.
+
+|===
+| Variable name | Description | Default value
+
+|repo
+|Repository to install JDK.
+|rhel-7-server-optional-rpms
+
+|java_version
+|JDK version to install.
+|1.8.0
+
+|===
+
+
+=== login
+
+|===
+| Variable name | Description
+
+|oc_user
+|OpenShift user to login to OpenShift.
+
+|oc_password
+|OpenShift user password.
+
+|===
+
+=== provision-osx
+
+Variables below are used while provisioning an OSX node.
+
+|===
+| Variable name | Description | Default value
+
+|remote_tmp_dir
+|What directory to use when creating some temporary files.
+|/tmp
+
+|nvm_install_url
+|The URL of nvm installation script.
+|https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh
+
+|nvm_install_file_name
+|What to name the nvm installation script on the node.
+|install-nvm.sh
+
+|rvm_install_url
+|What to name the rvm installation script on the node.
+|https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer
+
+|rvm_install_file_name
+|What to name the rvm installation script on the node.
+|install-rvm.sh
+
+|rvm_gpg_url
+|The URL of the rvm prerequisite gpg key.
+|https://rvm.io/mpapis.asc
+
+|rvm_gpg_file_name
+|What to name the rvm gpg file on the node.
+|mpapis.asc
```

Comment:
All rvm vars above can be removed as they will be dropped with this pr https://github.com/aerogear/digger-installer/pull/81